package.json

```diff
@@ -30,48 +30,49 @@
     "cli-table3": "^0.6.3",
     "devtools-protocol": "^0.0.1140464",
     "didyoumean": "^1.2.2",
-    "dotenv": "16.3.1",
+    "dotenv": "^16.4.5",
     "ejs": "^3.1.10",
     "envinfo": "7.11.0",
     "glob": "7.2.3",
     "jsdom": "^24.1.0",
     "lodash": "^4.17.21",
     "minimatch": "3.1.2",
-    "minimist": "1.2.6",
-    "mocha": "10.3.0",
+    "minimist": "^1.2.8",
+    "mocha": "^10.7.3",
     "nightwatch-axe-verbose": "^2.3.0",
     "open": "8.4.2",
     "ora": "5.4.1",
     "piscina": "^4.3.1",
-    "selenium-webdriver": "4.24.1",
+    "selenium-webdriver": "^4.25.0",
     "semver": "7.5.4",
     "stacktrace-parser": "0.1.10",
     "strip-ansi": "6.0.1",
+    "typescript-eslint": "^8.7.0",
     "untildify": "4.0.0",
-    "uuid": "8.3.2"
+    "uuid": "^10.0.0"
   },
   "devDependencies": {
-    "@cucumber/cucumber": "^8.2.1",
+    "@cucumber/cucumber": "^11.0.1",
```

Comment:
I have kicked off a ci run. I think this will break things

Reply:
After reviewing the issues caused by the recent updates to the project dependencies, I've decided to downgrade some package versions to restore stability to the project.

I ran a CI test, and here are the results:
```shell
➜ nightwatch git:(issue/3802-update-js-dependencies) npm ci
npm warn deprecated inflight@1.0.6: This module is not supported, and leaks memory. Do not use it. Check out lru-cache if you want a good and tested way to coalesce async requests by a key value, which is much more comprehensive and powerful.
npm warn deprecated @humanwhocodes/config-array@0.11.14: Use @eslint/config-array instead
npm warn deprecated rimraf@3.0.2: Rimraf versions prior to v4 are no longer supported
npm warn deprecated glob@7.2.3: Glob versions prior to v9 are no longer supported
npm warn deprecated glob@8.1.0: Glob versions prior to v9 are no longer supported
npm warn deprecated glob@8.1.0: Glob versions prior to v9 are no longer supported
npm warn deprecated @humanwhocodes/object-schema@2.0.3: Use @eslint/object-schema instead
npm warn deprecated reflect-metadata@0.2.1: This version has a critical bug in fallback handling. Please upgrade to reflect-metadata@0.2.2 or newer.

> nightwatch@3.8.0 prepare
> husky install

install command is DEPRECATED

added 882 packages, and audited 883 packages in 1m

189 packages are looking for funding
  run `npm fund` for details

6 vulnerabilities (5 high, 1 critical)

To address all issues possible (including breaking changes), run:
  npm audit fix --force

Some issues need review, and may require choosing
a different dependency.

Run `npm audit` for details.
```